<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusSaveData</class>
 <widget class="QDialog" name="MotusSaveData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="zongqingCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵倾</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hengyaoCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横摇</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hangxiangCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hengyiCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横移</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qianchongCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前冲</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shengjiangCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升降</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="datasaveCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveDataButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
